src/pages/Contact/Contact.js

```diff
@@ -1,32 +1,128 @@
+    }
+    &:focus {
+      outline: 0;
+      border-bottom: 3px solid #6831DE;
+    }
+  }
+  
+  textarea {
+      width: 100%;
+      border-style: solid;
+      border-width: 1px;
+  }
+
+  button,
+  input[type='submit'] {
+    width: auto;
+    background: linear-gradient(to bottom right, #E03BFB, #6831DE);;
+    color: white;
+    border: 0;
+    border-radius: 10px;
+    font-size: 1.2rem;
+    font-weight: 600;
+    text-transform: uppercase;
+    white-space: nowrap;
+    padding: 0.4rem 2.4rem;
+    cursor: pointer;
+  }
+
+  .btnContainer {
+    margin: 0 auto;
+    text-align: center;
+  }
+`
+
 const Contact = () => (
-    <div className="background">
-        <div className="Contact">
-            <form name="contact" method="POST">
-                <fieldset>
-                    <legend>Send us a Message!</legend>
-                    <div className="Contact-item">
-                        <label>Name</label>
-                        <br />
-                        <input type="text" name="name" />
-                    </div>
-                    <div className="Contact-item">
-                        <label>Email</label>
-                        <br />
-                        <input type="email" name="name" />
-                    </div>
-                    <div className="Contact-item">
-                        <label>Message</label>
-                        <br />
-                        <textarea type="text" rows="6" />
-                    </div>
-                </fieldset>
-            </form>
-        </div>
+    <div className="contactForm">
+        <Form>
+            <div className="header">
+                <h2>Contact Us</h2>
+                <p>If you have any enquiries please contact us by filling the form below:</p>
+            </div>
+            <label htmlFor="name">
+                First Name 
+                <input
+                    type="text"
+                    name="firstname"
+                    placeholder="Arnas"
+                    required
+                />
+            </label>
+            <label htmlFor="email">
+                Email
+                <input
+                    type="email"
+                    name="email"
+                    placeholder=""
+                    required
+                />
+            </label>
+            <label htmlFor="subject">
+                Subject 
+                <input
+                    type="text"
+                    name="subject"
+                    placeholder="Bug on homepage"
+                    required
+                />
+            </label>
+            <label htmlFor="message">
+                Your Message
+                <textarea 
+                    name="" 
+                    id="" 
+                    cols="30" 
+                    rows="10">
+                </textarea>
+            </label>
+            <div className="btnContainer">
+                <button type="submit">Send</button>
```

Comment:
On submit create a function that stores all info into state object. Instead of using a functional component, create a class. As well as don't allow the form to refresh the page using preventDefault();

Reply:
done! I now log form data and clear the form after submit.
```react
handleSubmit(event) {
    event.preventDefault();
    console.log(this.state);
    this.refs.form.reset();
} 
```